ai-integrations/llamaindex.ipynb

```diff
@@ -146,7 +137,7 @@
     "      {\n",
     "        \"type\": \"vector\",\n",
     "        \"path\": \"embedding\",\n",
-    "        \"numDimensions\": 1536,\n",
+    "        \"numDimensions\": 1024,\n",
```

Comment:
Should we add the polling code here to wait until the search index has built successfully before proceeding?